app/src/main/java/com/github/se/eventradar/model/repository/user/FirebaseUserRepository.kt

```diff
@@ -182,6 +183,29 @@ class FirebaseUserRepository(db: FirebaseFirestore = Firebase.firestore) : IUser
     }
   }
 
+  override suspend fun addAttendingEvent(userId: String, attendingEventId: String): Resource<Unit> {
+    return try {
+      userRef.document(userId).update("eventsAttendeeList", FieldValue.arrayUnion(attendingEventId))
+      Resource.Success(Unit)
+    } catch (e: Exception) {
+      Resource.Failure(e)
+    }
+  }
+
+  override suspend fun removeAttendingEvent(
```

Comment:
nit: likewise for this, maybe removeAttendeeFromEvent, but let me know what you think